hadoop-tools/hadoop-fs2img/src/main/java/org/apache/hadoop/hdfs/server/namenode/UGIResolver.java

```diff
@@ -132,4 +135,57 @@ public FsPermission permission(FileStatus s) {
     return s.getPermission();
   }
 
+  private long resolve(AclStatus aclStatus) {
```

Comment:
static?

Reply:
No. Static will not work as resolution methods in the subclass may not be static.